packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -116,7 +123,7 @@ export class AtomicSearchBox {
       id: this.id,
       searchBoxController: this.searchBox,
       numberOfQueries: this.numberOfQueries,
-      inputRef: this.inputRef,
```

Comment:
It took me way too long to notice this, but the reason that focus wasn't coming back to the search box after selecting a suggestion was because it was being explicitly blurred. I removed the blur and introduced a `clearSuggestions` function as a replacement, since I'm guessing this was the intended behaviour.